info/contributing.md

```diff
@@ -0,0 +1,16 @@
+![Contributing To Torrust Title Image](../img/2022_06_27-Contributing_To_TorrustTitle.png)
+
+# Contribution Guide for the Torrust Project
+
+__Thank You__ for your interest in our project, we always strive to make the onboarding process for new contributors as clear and comfortable as possible.
+
+To ensure that the project continues to improve without compromising its integrity and quality, we kindly ask that you follow this guide to help ensure that your contributions are harmonic as possible.
```

Comment:
your contributions are harmonic as possible.    ---- Forgot one "as"
Change to:
your contributions are as harmonic as possible.